docs/api/menu.md

```diff
@@ -80,6 +80,10 @@ The `menu` object has the following instance methods:
   * `positioningItem` number (optional) _macOS_ - The index of the menu item to
     be positioned under the mouse cursor at the specified coordinates. Default
     is -1.
+  * `initiatedByKeyboard` boolean (optional) _Windows_ _Linux_ - Whether the menu popup
+    was triggered by keyboard actions.  This will cause certain focus behaviors to change
+    so that the first item in the menu is focused when the menu opens which is beneficial
+    for keyboard accessibility.
```

Comment:
docs: I didn't really like the "certain focus behaviors" wording so here's an attempt at being more precise.

```suggestion
  * `initiatedByKeyboard` boolean (optional) _Windows_ _Linux_ - Whether the menu popup
    was triggered by a keyboard action. If true, the first top-level item in the menu
    will be focused when the menu is opened. This behavior can assist in keyboard
    accessibility.
```

Reply:
api: Do we want to make this API something like `focusFirstItem` instead?

It seems a bit counter-intuitive that "Initiated By Keyboard" is exposed as an API option for the developer to wire up in their JavaScript code rather than something that's auto-detected.

Also, do we ever foresee using any of the other [MenuSourceType](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/ui_base_types.h;bpv=1;bpt=1;l=133?q=MENU_SOURCE_KEYBOARD&ss=chromium&gsn=MenuSourceType&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fchromium.googlesource.com%2Fcodesearch%2Fchromium%2Fsrc%2F%2Fmain%3Flang%3Dc%252B%252B%3Fpath%3Dui%2Fbase%2Fui_base_types.h%23MenuSourceType%253Aui%2523n%2523TX9FKDk8TS) options available?

Reply:
`focusFirstItem` isn't the _only_ effect this has hence the matching of the "keyboard" name

Reply:
> Also, do we ever foresee using any of the other [MenuSourceType](https://source.chromium.org/chromium/chromium/src/+/main:ui/base/ui_base_types.h;bpv=1;bpt=1;l=133?q=MENU_SOURCE_KEYBOARD&ss=chromium&gsn=MenuSourceType&gs=KYTHE%3A%2F%2Fkythe%3A%2F%2Fchromium.googlesource.com%2Fcodesearch%2Fchromium%2Fsrc%2F%2Fmain%3Flang%3Dc%252B%252B%3Fpath%3Dui%2Fbase%2Fui_base_types.h%23MenuSourceType%253Aui%2523n%2523TX9FKDk8TS) options available?

Lots of these are legacy / not used anymore so no not really